physical/mongo_test.go

```diff
@@ -0,0 +1,58 @@
+package physical
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/vault/helper/logformat"
+	log "github.com/mgutz/logxi/v1"
+)
+
+func TestMongoBackend(t *testing.T) {
+	logger := logformat.NewVaultLogger(log.LevelTrace)
+
+	mongoUri := os.Getenv("MONGODB_URI")
```

Comment:
ok, it was all very confusing to me how the tests are actually running ... thanks for pointing that out to me! I will dig into that